src/lib/hooks/swap/useSwapInfo.tsx

```diff
@@ -121,13 +123,16 @@ export default function useSwapInfo(): SwapInfo {
   // swapInfo will lag behind swapAtom by a frame, because its update is triggered by swapAtom
```

Comment:
do we need this comment still if context is merged? Also does that change remove need to do this

Reply:
We will not need this comment after #3710 is merged, but #3710 removes it (and all of the logic in this hook).

Reply:
great